spec/rubocop/cop/style/numeric_predicate_spec.rb

```diff
@@ -188,10 +188,20 @@
       {
         'EnforcedStyle' => 'predicate',
         'AutoCorrect' => true,
-        'IgnoredMethods' => %w[where]
+        'IgnoredMethods' => %w[== where]
```

Comment:
That's not the case discussed in #7595, which has `zero?` in the ignored methods. I guess the reporter's expectation was that he could prevent the _introduction_ of this method as well by adding it to the ignored ones.

Reply:
Thanks for the feedback. I didn't mention my thought on this PR fix in the issue 💦 I added a comment to Issue https://github.com/rubocop-hq/rubocop/issues/7595#issuecomment-569001517 and updated this test case accordingly.

Reply:
Thanks. Those thoughts were missing indeed.